apps/condo/domains/common/components/settings/CommonSettingsContent.tsx

```diff
@@ -0,0 +1,129 @@
+import { Table } from 'antd'
+import { ColumnsType } from 'antd/es/table'
+import React, { useCallback, useMemo, useState } from 'react'
+
+import { LocaleContext, useIntl } from '@open-condo/next/intl'
+import { Button, Modal, Radio } from '@open-condo/ui'
+
+interface ISettingsPopupContentProps<T> {
+    value: T
+    onChange: (value: T) => void
+}
+
+interface ILanguageSettingPopupContentProps extends ISettingsPopupContentProps<string> {
+    possibleLocales: { [locale: string]: string }
+}
+
+interface ISettingsTableItem {
+    title: string
+    valueCell: JSX.Element
+    modalContentComponent: React.FC<ISettingsPopupContentProps<unknown>>
+    value: unknown
+    onChange: { (data: unknown): void }
+    props: { [key: string]: unknown }
+    onSave: { (data: unknown): void }
+}
+
+type TCurrentSettingModalData = Omit<ISettingsTableItem, 'valueCell'>
+
+const LanguageSettingPopupContent: React.FC<ILanguageSettingPopupContentProps> = ({
+    value,
+    onChange,
+    possibleLocales,
+}) => {
+    return (
+        <div>
+            {Object.entries(possibleLocales).map(([locale, localeName]) => (
+                <div key={`locale=${locale}`}>
+                    <Radio
+                        label={localeName}
+                        value={locale}
+                        checked={locale === value}
+                        onChange={(event) => onChange(event.target.value)}
+                    />
+                </div>
+            ))}
+        </div>
+    )
+}
+
+export const CommonSettingsContent: React.FC = () => {
```

Comment:
Strange naming for me. Will this tab be used for anything other than language selection?

Reply:
For now, there is only one setting - the language. Maybe we will add something else or move this option to the user's profile in the future.